features/step_definitions/mobile_steps.rb

```diff
@@ -33,3 +33,7 @@
 When /^I open the drawer$/ do
   find('#menu_badge').click
 end
+
+Then /^the aspect dropdown should be labeled "([^"]*)"/ do |label|
+  expect(page).to have_css("#user_aspects option.list_cover", :text => label)
```

Comment:
Oh and btw., I wouldn't have any objections to have equivalents to https://github.com/diaspora/diaspora/blob/develop/features/step_definitions/aspects_steps.rb#L55 etc. for the mobile version of the aspect dropdown.